Atomic/util.py

```diff
@@ -1080,3 +1081,11 @@ def set_proxy():
 class ImageAlreadyExists(Exception):
     def __init__(self, img):
         super(ImageAlreadyExists, self).__init__("The latest version of image {} already exists.".format(img))
+
+
+def kpod(cmd, storage=None, debug=None):
```

Comment:
tests are failing since debug is not used here

Reply:
yup, fixed!